js/interface.js

```diff
@@ -62,36 +62,39 @@ function parseThumbnail(file) {
 
 function navigateToDefaultFolder() {
   if (typeof data === 'undefined' || !data || !data.appId) {
-    // No folder was specified
```

Comment:
why did you change the indentation on these comments?

Reply:
done